presto-raptor/src/test/java/com/facebook/presto/raptor/metadata/TestRaptorMetadata.java

```diff
@@ -448,6 +454,52 @@ public void testListTableColumnsFiltering()
     }
 
     @Test
+    public void testTableIdentity()
+            throws Exception
+    {
+        // Test TableIdentity round trip.
+        metadata.createTable(SESSION, getOrdersTable());
+        ConnectorTableHandle connectorTableHandle = metadata.getTableHandle(SESSION, DEFAULT_TEST_ORDERS);
+        TableIdentity tableIdentity = metadata.getTableIdentity(connectorTableHandle);
+        byte[] bytes = tableIdentity.serialize();
+        assertEquals(tableIdentity, metadata.deserializeTableIdentity(bytes));
+
+        // Test one hard coded serialized data for each version.
+        byte version = 1;
+        long tableId = 12345678L;
+        ByteArrayDataOutput dataOutput = ByteStreams.newDataOutput(Byte.BYTES + Long.BYTES);
```

Comment:
Since this is a test, just use `newDataOutput()`. Also, static import `newDataOutput`